src/lib/components/Button.svelte

```diff
@@ -62,8 +62,8 @@
 	.bg-gold {
 		background-color: $gold;
 	}
-	.bg-earth-green {
-		background-color: $earth-green;
+	.bg-green {
```

Comment:
Looks like you use lime in a lot of other places besides here, maybe keep the naming consistent and name the selector `.bg-lime`.